water/test/Transforms/lowered_gemm_pipelined.mlir

```diff
@@ -0,0 +1,126 @@
+// RUN: water-opt %s --allow-unregistered-dialect
+
+module attributes {transform.with_named_sequence} {
```

Comment:
Why do we need this mlir? If its just to check if water-opt works on unregistered dialects should we choose something simpler?

Reply:
This is an integration-style test to check registration of all relevant dialect, lack of registration for irrelevant dialect and generally break if something relevant in LLVM changes.